components/extensions/vscode/src/Commands.ts

```diff
@@ -19,12 +19,15 @@
 import * as path from "path";
 import * as vscode from "vscode";
 import Constants from "./constants";
+import CommandsUtils from "./utils/CommandsUtils";
 
 /**
  * Cellery commands.
  */
 class Commands {
     private static readonly terminals: { [name: string]: vscode.Terminal } = { };
+    private static readonly imageNames: string[] = ["add image name"];
```

Comment:
Let's rename this to something a bit more meaningful. (Same for instance names)

Suggestion:- cachedImageNames

Also let's move the "add image name" to a separate constant and remove it from this array. We should add it only when it is required for the sake of meaningfulness.